res/controllers/Numark-Mixtrack-Platinum-FX-scripts.js

```diff
@@ -0,0 +1,1899 @@
+// eslint-disable-next-line no-var
+var MixtrackPlatinumFX = {};
+
+// FX toggles
+MixtrackPlatinumFX.toggleFXControlEnable = true;
+MixtrackPlatinumFX.toggleFXControlSuper = false;
+
+MixtrackPlatinumFX.shifBrowseIsZoom = false;
+
+// setting this to false sets tap the file bpm, but without a way to reset its dangerous
+MixtrackPlatinumFX.tapChangesTempo = true;
+
+// pitch ranges
+// add/remove/modify steps to your liking
+// default step must be set in Mixxx settings
+// setting is stored per deck in pitchRange.currentRangeIdx
+MixtrackPlatinumFX.pitchRanges = [0.08, 0.16, 0.5];
+
+MixtrackPlatinumFX.HIGH_LIGHT = 0x7F;
+MixtrackPlatinumFX.LOW_LIGHT = 0x01;
+
+// whether the corresponding Mixxx option is enabled
+// (Settings -> Preferences -> Waveforms -> Synchronize zoom level across all waveforms)
+MixtrackPlatinumFX.waveformsSynced = true;
```

Comment:
why is that relevant for the mapping?

Reply:
// need to zoom both channels if waveform sync is disabled in Mixxx settings.
                                // and when it's enabled then no need to zoom 2nd channel, as it will cause
                                // the zoom to jump 2 levels at once

Reply:
I see I see. I think this would be better solved by introducing a CO that always zooms the CO in unison regardless of the preference setting. Can you file a feature request for that?